specs/requirements.md

```diff
@@ -14,8 +14,9 @@ For each point prepended by priority:
 ### Functionality
 
 1. Camera devices
-    * Setup workflow
+    * Provide initialization interface
         * [1] Register API authorization key on device
+        * [1] Make test API call
         * [2] Enable/disable logging to device
         * [3] Provide setup GUI
```

Comment:
How would you argue that it's not in non-functional part (as we were askeded to move it there but I would also leave it here)

Reply:
One can say you have to provide setup script for the user.